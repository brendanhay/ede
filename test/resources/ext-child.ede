{}
---
{% extends "ext-base.ede" %}
{% block title %}Index{% endblock %}
{% block head %}
  {{ super }}
  <style type="text/css">
    .important { color: #336699; }
  </style>
{% endblock %}
{% block body %}
  <h1>Index</h1>
  <p class="important">
    Welcome to my awesome homepage.
  </p>
{% endblock %}