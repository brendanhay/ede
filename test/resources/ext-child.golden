{}
---
<!doctype html>
  <title>Index</title>

  <style type="text/css">
    .important { color: #336699; }
  </style>
  <h1>Index</h1>
  <p class="important">
    Welcome to my awesome homepage.
  </p>

